fake_switches/brocade/command_processor/enabled.py

```diff
@@ -67,7 +67,7 @@ def do_skip_page_display(self, *args):
         pass
 
     def do_write(self, *args):
-        pass
+        self.switch_configuration.commit()
```

Comment:
```
self.switch_configuration.commit()
```

When I read this, it feels like I'm about to commit the switch configuration....  I would prefer to see
```
sleep(self.switch_configuration.commit_delay)
```
or
```
self.switch_configuration.simulate_commit_delay()
```